testgrid/config/config.yaml

```diff
@@ -895,13 +895,6 @@ test_groups:
   gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-e2e-gce-garbage
 - name: ci-kubernetes-e2e-gci-gce-garbage
   gcs_prefix: kubernetes-jenkins/logs/ci-kubernetes-e2e-gci-gce-garbage
-# kubeadm tests
```

Comment:
I think this is the wrong place, but not sure. The GCE dashboard seems to be somewhere else or?